src/js/server/hcapi/applications.js

```diff
@@ -1,37 +1,163 @@
 const { Router } = require('express');
-const { Hacker, HackerApplication, ApplicationResponse } = require('js/server/models');
+const Sequelize = require('sequelize');
+const { Hacker, HackerApplication, ApplicationResponse, TeamMember, Team, db } = require('js/server/models');
 const { createHttpError } = require('./errors');
+const fs = require('fs');
+const individualScoreQuery = fs.readFileSync('src/js/server/hcapi/queries/indiv_score_unteamed.sql', 'utf8');
 
 const applicationsRouter = new Router();
 
+// Takes an application and the set of all individual scores and
+// the set of teams average scores
+// returns the score for that application as a float
+function calculateScore(application, individualScores, teamScores) {
+  let hacker = application.hacker;
+  let teamId = hacker.Team ? hacker.Team.teamId : null;
+
+  if (teamId == null) {
+    // The hacker isn't in a team
+    if (application.inTeam) {
+      // If they applied as a team then their application is incomplete, return null
+      return null
```

Comment:
Do we want to keep this check? After applications are actually closed, all bets are off as to whether these people will actually apply in a team